### PR TITLE
Publish NER blog

### PR DESCRIPTION
**Pull request by smoreface**

### CODE REVIEW

File: docs/source/blog/nereval.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+type: blog
+image: ![image](https://user-images.githubusercontent.com/2641205/112475305-f5863900-8d70-11eb-9eb8-9da879f61708.png)
+order: 96
+meta_title: Evaluate NER parsers with spaCy and Label Studio
+meta_description: Use Label Studio to evaluate named entity recognition parsers like spaCy and revise predictions by annotating a gold standard dataset for your data science and machine learning projects. 
+---
+
+![image](https://user-images.githubusercontent.com/2641205/112475305-f5863900-8d70-11eb-9eb8-9da879f61708.png)
+
+This tutorial helps you evaluate accuracy of Named Entity Recognition (NER) taggers using Label Studio. Gather predictions from standard [spaCY](https://spacy.io/) language models for a dataset based on transcripts from the podcast This American Life, then use Label Studio to correct the transcripts and determine which model performed better to focus future retraining efforts.
```
Comment (niklub) [inline docs/source/blog/nereval.md:12]: should we provide the link to the podcast transcripts here?

File: docs/source/blog/nereval.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    f.write('\n'.join(sorted(entities)))
+```
+
+After running the script, you have two files:
+- A `tasks.json` file with predictions from the large and small spaCy models to import into Label Studio.
+- A `named_entities.txt` file that contains the list of entities to use as labels.
+ 
+## Correct the predicted Named Entities in Label Studio
+
+To classify named entities, you need to create a dataset with gold standard labels that are accurate for your use case. To do that, use the open source data labeling tool, [Label Studio](https://labelstud.io).
```
Comment (niklub) [inline docs/source/blog/nereval.md:149]: do we actually need the link here?

File: docs/source/blog/index.html [hunk 3]
```diff
@@ -138,6 +138,20 @@
...
 
+        <!-- Release 1.0.0 -->
+    <div class="column">
+      <a href="/blog/nereval.html">
+        <div class="card">
+          <div class="image-wrap">
+            <div style="background-image: url(https://user-images.githubusercontent.com/2641205/112475305-f5863900-8d70-11eb-9eb8-9da879f61708.png); background-size:cover" class="image"></div>
+          </div>
+          <div class="category">article</div>
+          <div class="desc">3 May 2021, 4 min read</div>
```
Comment (niklub) [inline docs/source/blog/index.html:149]: need some realistic estimates here

**Review by niklub — CHANGES_REQUESTED** [review]

File: docs/source/blog/nereval.md (hunk 1 above)
Comment (smoreface) [reply docs/source/blog/nereval.md:11]: It's in the steps later, I don't want to distract people up here

File: docs/source/blog/nereval.md (hunk 2 above)
Comment (smoreface) [reply docs/source/blog/nereval.md:148]: no, will remove

**Review by niklub — APPROVED** [review]

**Pull request merged**